code/_onclick/click_override.dm

```diff
@@ -71,7 +71,7 @@
 	var/atom/beam_from = user
 	var/atom/target_atom = A
 
-	for(var/i in 0 to 3)
+	for(var/i in 0 to 2)
```

Comment:
This feels like a magic number. Can you make a define or otherwise document this?